### PR TITLE
53 nationcred self activation

### PR DESCRIPTION
**Pull request by johnmark13**
Now with almost no changes. Updated Readme, updated packages to use latest versions, updated config (still has my test server in, can be removed once deployed to Nation3 Heroku)

Test command will write updates to the nation3/nationcred-instance activation-bot-test branch, that branch should not be merged, but you can check the behaviour of the Discord commands there.

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -1,10 +1,18 @@
 module.exports = {
     GUILD_IDS: {
-        "798975165342023700": {
-            repo: "sourcecred/dev-cred", 
-            branch: "payout-bot-test", 
+        "956551108061446244": {
+            repo: "nation3/nationcred-instance", 
+            branch: "activation-bot-test", 
+            discordBaseURL: "https://forum.nation3.org",
```
Comment (aahna-ashina) [inline config.js]: @johnmark13 Why is the `discordBaseURL` a link to the forum?

**Review by aahna-ashina — COMMENTED** [review]
Would it be easy to set a custom commit message?  For example "Activated JohnMark13"?

Screen Shot 2022-06-02 at 4 48 09 PM

File: config.js (hunk 1 above)
Comment (johnmark13) [reply config.js]: Actually, now we removed Discourse we do not need that at all, so I will remove, good spot.

**Comment by johnmark13** [comment]
> Would it be easy to set a custom commit message? For example "Activated JohnMark13"?
> 
> Screen Shot 2022-06-02 at 4 48 09 PM

Yeah, this is a great idea, but I cannot see a way to achieve it (looking at source) - https://github.com/sourcecred/sourcecred/blob/a6e1d82216da4a2d909aaeb6c908696bc692b32a/packages/sourcecred/src/api/ledgerManager.js#L77-L104

**Review by aahna-ashina — APPROVED** [review]

**Pull request merged**